### PR TITLE
Added documentation for Sector Alarm component

### PR DESCRIPTION
**Pull request by peroyvind**
**Description:**
Documentation for the new Sector Alarm component

## Checklist:

- [X] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [X] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/sectoralarm.markdown
Comment (klaasnicolaas) [inline source/_components/sectoralarm.markdown]: You can remove line 15 - 17

File: source/_components/sectoralarm.markdown
Comment (klaasnicolaas) [inline source/_components/sectoralarm.markdown:37]: Please remove the whitespaces between the variables.

File: source/_components/sectoralarm.markdown
Comment (klaasnicolaas) [inline source/_components/sectoralarm.markdown]:
```suggestion
  email: EMAIL_ADDRESS
```

File: source/_components/sectoralarm.markdown
Comment (klaasnicolaas) [inline source/_components/sectoralarm.markdown]:
```suggestion
  password: PASSWORD
```

File: source/_components/sectoralarm.markdown
Comment (klaasnicolaas) [inline source/_components/sectoralarm.markdown]:
```suggestion
  code: ALARM_CODE
```

File: source/_components/sectoralarm.markdown
Comment (klaasnicolaas) [inline source/_components/sectoralarm.markdown]:
```suggestion
  alarm_id: ALARM_ID
```

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

**Review by klaasnicolaas — APPROVED** [review]
✅ Can be merged after merging the parent PR 👍

**Comment by klaasnicolaas** [comment]
Parent PR still open

**Comment by frenck** [comment]
Closing PR, since the parent PR has been closed.

**Pull request closed without merging**
